### PR TITLE
Updated whole Spanish translations

### PR DESCRIPTION
**Pull request by JaviG1105**
The entire Spanish segment translations have been added.

### CODE REVIEW

**Comment by delphiactual** [comment]
This is great work

File: app/scripts/dimApp.i18n.js [hunk 1]
```diff
@@ -545,36 +545,193 @@
...
+            CharacterOrderRecent: "Por el personaje más reciente",
+            CharacterOrderReversed: "Por el personaje más reciente (Reverso)",
+            CharacterOrderFixed: "Fijo (Por la edad del personaje)",
+            ExportSS: "Descargar hojas de cálculo",
+            ExportSSHelp: "Descargar una lista CSV de tus objetos que puede ser vista fácilmente en una aplicación de hojas de cálculo de tu preferencia.",
+            DIMPopups: "Ventanas emergentes de información de DIM",
+            DIMPopupsReset: "Reestablecer tips de información anteriormente escondidos" },
           Stats: {
             Intellect: "Intelecto",
             Discipline: "Disciplina",
```
Comment (delphiactual) [inline app/scripts/dimApp.i18n.js]: NoBonus is missing

File: app/scripts/dimApp.i18n.js [hunk 2]
```diff
@@ -545,39 +545,197 @@
...
+            AppliedError: "Ninguno de los objetos en tu equipo pudo ser transferido.",
+            AppliedWarn: "Tu equipo ha sido parcialmente transferido, pero {{failed}} de {{total}} objetos tuvieron errores." },
+          Manifest: {
+            Build: "Creando información de la base de datos de Destiny",
+            Download: "Descargando información más actualizada de Destiny desde Bungie",
+            Error: "Error cargando Destiny info:\n{{error}}\n Carga de nuevo para volver a intentarlo.",
+            Outdated: "Información de Destiny obsoleta",
+            OutdatedExplanation: "Bungie ha actualizado su información de base de datos de Destiny. Recarga DIM para cargar la información nueva. Toma en cuenta que puede que DIM no funcione por algunas horas después de que Bungie actualiza Destiny, mientras que los nuevos datos se propagan por sus sistemas.",
+            BungieDown: "Bungie.net puede estar teniendo problemas.",
+            Load: "Cargando información de Destiny guardada",
```
Comment (delphiactual) [inline app/scripts/dimApp.i18n.js]: Can you please add LoadCharInv: "Loading Destiny characters and inventory", here with the correct translation, do not add it to the English definitions though, it is a new translation that will automatically be applied at a later date.

Comment (JaviG1105) [inline app/scripts/dimApp.i18n.js]: Sure thing.

**Comment by delphiactual** [comment]
g2g 🚢

**Comment by kyleshay** [comment]
awesome!! thank you so much both of you!

**Pull request merged**